packages/ui/components/form/wizard/WizardForm.tsx

```diff
@@ -37,7 +39,7 @@ function WizardForm<T extends DefaultStep>(props: {
       <img className="mx-auto mb-8 h-8" src="https://cal.com/logo.svg" alt="Cal.com Logo" />
       <div
         className={classNames(
-          "mb-8 divide-y divide-gray-200 overflow-hidden rounded-lg bg-white shadow print:divide-transparent print:shadow-transparent",
+          "mb-8 w-[700px] divide-y divide-gray-200 overflow-hidden rounded-lg bg-white shadow print:divide-transparent print:shadow-transparent",
```

Comment:
Wizard was extending way more in width that it should, so limiting it from here.

Reply:
Should this be max width instead of a fixed one?

Reply:
Yes - this will likely break on mobile without max width